app/build.gradle

```diff
@@ -13,11 +13,8 @@ android {
         versionName "1.0"
         testInstrumentationRunner "android.support.test.runner.AndroidJUnitRunner"
 
-        def twitter_consumer_key = hasProperty('twitter_consumer_key') ? twitter_consumer_key : System.getenv('TWITTER_CONSUMER_KEY')
-        def twitter_consumer_secret = hasProperty('twitter_consumer_secret') ? twitter_consumer_secret : System.getenv('TWITTER_CONSUMER_SECRET')
-
-        buildConfigField "String", "TWITTER_CONSUMER_KEY", '"' + twitter_consumer_key + '"'
-        buildConfigField "String", "TWITTER_CONSUMER_SECRET", '"' + twitter_consumer_secret + '"'
+        buildConfigField "String", "TWITTER_CONSUMER_KEY", project.hasProperty("twitter_consumer_key") ? '"' + project.twitter_consumer_key + '"' : '"' + System.getenv('TWITTER_CONSUMER_KEY') + '"'
```

Comment:
Another option is to always use the gradle properties and forget about the environment variables.

You can write the environment variables like this: `ORG_GRADLE_PROJECT_ twitter_consumer_key` and gradle will convert it to a property named `twitter_consumer_key`.

Reply:
Thanks for your comment, changed ;)